app/lib/feed.rb

```diff
@@ -0,0 +1,69 @@
+class Feed
+  def initialize(educator)
+    @educator = educator
+    @authorizer = Authorizer.new(@educator)
+  end
+
+  # Return a list of `FeedCard`s for the feed based on event notes.
+  def event_note_cards(from_time, limit)
+    event_notes = @authorizer.authorized do
```

Comment:
Talking this out with @kevinrobinson, maybe we want to pass the event notes as a straight argument?